### PR TITLE
Pm 2022 02 styling issues

### PR DESCRIPTION
**Pull request by phillimorland**

### CODE REVIEW

File: apps/home/templates/apps_home/blocks/teaser_block_two_columns.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
 <div class="modul-servicepanel fullwidth ds-block {% if self.background_color %}ds-block--{{ self.background_color }}{% else %}ds-block__body-bg{% endif %}">
     <div class="servicepanel__main flexgrid grid--2 text-center">
         <div class="">
```
Comment (khamui) [inline apps/home/templates/apps_home/blocks/teaser_block_two_columns.html:7]: empty class atrribute, should be removed?

File: digitalstrategie/templates/includes/footer.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
 {% load_site_menu "footer" as footer_menu_items %}
 
 {% if footer_menu_items %}
-<div id="content-footer" role="contentinfo">
+<div class="content-footer" id="content-footer" role="contentinfo">
     <div class="content-footer__brand">
```
Comment (khamui) [inline digitalstrategie/templates/includes/footer.html:6]: is this a placeholder for something? or can be removed?

File: apps/home/templates/apps_home/blocks/teaser_block_two_columns.html (hunk 1 above)
Comment (phillimorland) [reply apps/home/templates/apps_home/blocks/teaser_block_two_columns.html:7]: this block will likely be deleted when we get final go ahead on designs so I thought to just leave it for now

File: digitalstrategie/assets/scss/_base.scss [hunk 3]
```diff
@@ -22,6 +22,7 @@ body {
 }
 
 header,
-footer {
+footer,
+.content-footer {
```
Comment (phillimorland) [inline digitalstrategie/assets/scss/_base.scss:26]: @khamui in answer to youre place holder question

**Comment by khamui** [comment]
cool!

**Pull request merged**